css/ampdoc.css

```diff
@@ -774,61 +293,14 @@ amp-experiment, amp-share-tracking {
   padding: 5px;
 }
 
-/**
```

Comment:
there are bunch of `amp-live-list`, `amp-list`, `amp-autocomplete` and `amp-fx-collection` rules that I think shuld go in element as well